docs/source/release_notes.rst

```diff
@@ -2,13 +2,25 @@ Release Notes
 -------------
 **Future Releases**
     * Enhancements
-        * Transfer over base data checks and "IDColumnData Checks" from the EvalML repo :pr:`1`
+        * Added in github workflows that are relevant to DataChecks, from EvalML repository, and modified to fit DataChecks wherever possible :pr:`2`
+        * Implemented linters and have them successfully running :pr:`2`
     * Fixes
-        * Fixed pytest failures :pr:`1`
+        * Workflows are now up and running properly :pr:`2`
     * Changes
-        * Removed all EvalML dependencies and unnecessary functions/comments from utils, tests, exceptions, and data checks :pr:`1`
+        * Updated comments to reflect DataChecks repository :pr:`2`
+        * Restructured pdm packaging to only be relevant to DataChecks, now to be renamed to CheckMate :pr:`2`
     * Documentation Changes
     * Testing Changes
+        * Removed integration testing due to irrelevance with base datachecks and id column data check :pr:`2`
```

Comment:
If we're referencing files or objects, let's surround them with backticks so they're formatted as code. For example `DataChecks`, `ID Columns Data Check`, etc

Reply:
Just added the backticks